index.js

```diff
@@ -1,14 +1,10 @@
 'use strict'
 
 var fs = require('fs')
-var Merge = require('merge-stream')
 var http = require('http')
 var qs = require('querystring')
 var multiaddr = require('multiaddr')
-var File = require('vinyl')
-var vinylfs = require('vinyl-fs-that-respects-files')
-var vmps = require('vinyl-multipart-stream')
-var stream = require('stream')
+var multipartFileStream = require('./files')
```

Comment:
Keeping consistency with names helps for code readability.

Since only one function is export with the name `fileStream`:
- 'files.js' -> 'files-stream.js'
- 'var multipartFileStream = require('./files')' -> var filesStream = require('./files-stream')